mods/ts/weapons/energyweapons.yaml

```diff
@@ -137,7 +136,6 @@ LaserFire:
 		Damage: 250
 		DamageTypes: Prone60Percent, TriggerProne, EnergyDeath
 
-# Laser turret
```

Comment:
This comment should be kept or `LaserFire2` should be renamed to `LaserTurret`[`Fire`].